liblwgeom/lwkmeans.c

```diff
@@ -157,11 +154,16 @@ kmeans_init(POINT2D** objs, int* clusters, uint32_t n, POINT2D** centers, POINT2
 		dst_p2 = distance2d_sqr_pt_pt(objs[i], objs[p2]);
 		if ((dst_p1 > max_dst) || (dst_p2 > max_dst))
 		{
-			max_dst = fmax(dst_p1, dst_p2);
 			if (dst_p1 > dst_p2)
+			{
```

Comment:
Are these changes related to the inlining?

Reply:
yes, they remove a call to fmax/fmin.